planetarium/NineChronicles
# Display indicator on start of crystal related actions.

### Description

1. Display indicator on start of crystal related actions.
2. Change button text of `PaymentPopup` when crystal is not enough.

### How to test

1. Check if indicator is properly displaying on start of `Grinding`, `CombinationEquipment`, `UnlockEquipment` and `ItemEnhancement` action.

### Required Reviewers
@planetarium/9c-dev @jaeho0103
